util/src/hex_tools.cpp

```diff
@@ -65,4 +65,30 @@ std::string sliverToHex(const Sliver &sliver) { return bufferToHex(sliver.data()
 
 std::string vectorToHex(const std::vector<std::uint8_t> &data) { return bufferToHex(data.data(), data.size()); }
 
+std::vector<uint8_t> unhex(const std::string &hex) {
+  const auto inputLen = hex.size();
```

Comment:
use boost::algorithm::unhex

Reply:
Changes in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2